registry.json

```diff
@@ -969,6 +969,33 @@
       "documentation": "https://polkadot.js.org/api/substrate/rpc.html"
     }
   },
+  {
+    "id": "everscale",
+    "name": "Everscale",
+    "coinId": 396,
+    "symbol": "EVER",
+    "decimals": 9,
+    "blockchain": "Everscale",
+    "derivation": [
+      {
+        "path": "m/44'/396'/0'/0/0"
+      }
+    ],
+    "curve": "ed25519",
+    "publicKeyType": "ed25519",
+    "explorer": {
+      "url": "https://everscan.io",
+      "txPath": "/transactions/",
+      "accountPath": "/accounts/",
+      "sampleTx": "781238b2b0d15cd4cd2e2a0a142753750cd5e1b2c8b506fcede75a90e02f1268",
+      "sampleAccount": "0:d2bf59964a05dee84a0dd1ddc0ad83ba44d49719cf843d689dc8b726d0fb59d8"
+    },
+    "info": {
+      "url": "https://everscale.network/",
+      "source": "https://github.com/broxus/ton-wallet-api",
+      "documentation": "https://tonapi.broxus.com/ton/v3/swagger.yaml"
```

Comment:
There were multiple public GQL API endpoints ([eri01.main.everos.dev/graphql](https://eri01.main.everos.dev/graphql), [gra01.main.everos.dev/graphql](https://gra01.main.everos.dev/graphql), [lim01.main.everos.dev/graphql](https://lim01.main.everos.dev/graphql)), but starting from September 1st they will only be available for the authorized users (e.g. https://mainnet.evercloud.dev/89a3b8f46a484f2ea3bdd364ddaee3a3/graphql)

Because of this, we first specified an alternative API here, but it’s better to use official even if it will require the creation of a project id.